WeScan/Edit/EditScanViewController.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 import AVFoundation
 
 /// The `EditScanViewController` offers an interface for the user to edit the detected quadrilateral.
-final class EditScanViewController: UIViewController {
+public class EditScanViewController: UIViewController {
```

Comment:
What is the exact reason for making this public?

Reply:
Oh, I think I forgot to change it back into final. When I try to use inherit inserted but It doesn't work. 😅